Source/missiles.cpp

```diff
@@ -30,6 +30,7 @@ int AvailableMissiles[MAXMISSILES];
 Missile Missiles[MAXMISSILES];
 int ActiveMissileCount;
 bool MissilePreFlag;
+constexpr int MSOURCE_TRAP = -1;

```

Comment:
Maybe `MSOURCE_OBJECT` or `MSOURCE_WORLD` would be better names like you suggested.

But also this should be used where the value is being set in the first place, not just for the check.

Reply:
Got the first part. Not sure what you mean for the second part. What am I missing?